packages/adapters/subgraph/src/lib/subgraphs/runtime/queries.ts

```diff
@@ -250,116 +145,57 @@ export const getExecutedTransfersByIds = gql`
   }
 `;
 
-export const getReconciledTransfersByIds = gql`
-  query GetReconciledTransfersByIds($transferIds: [Bytes!], $maxReconciledBlockNumber: BigInt!) {
-    transfers(
+export const getDestinationTransfersByIds = gql`
+  query GetDestinationTransfersByIds(
+    $transferIds: [Bytes!]
+    $maxExecutedBlockNumber: BigInt
+    $maxReconciledBlockNumber: BigInt
+    $status: TransferStatus
```

Comment:
Why specify status if you're using IDs anyways?

Reply:
it's an optional param for functionality which we were using in one of the query

Reply:
yeah, if we don't add `!` as suffix, then it's an optional param. Using this in code so it's all gucci